public/locales/en-GB/app.json

```
@@ -132,6 +132,7 @@
     "developer_settings_label": "Developer Settings",
     "developer_settings_label_description": "Expose developer settings in the settings window.",
     "developer_tab_title": "Developer",
+    "duplicate_tiles_label": "Number of duplicate tiles",
```

I think we can phrase this less technical (even though its in the developer settings)

```suggestion
    "duplicate_tiles_label": "Number of additional tiles copies per participant.",
```